python/dolfin/fem/__init__.py

```diff
@@ -9,6 +9,10 @@
 from dolfin.jit.jit import ffc_jit
 import dolfin.cpp
 
+from .form import Form
```

Comment:
Not sure about relative imports. I don't mind, but I think we reached an earlier consensus to avoid.